Add capability to read HCHO

blychs opened the pull request:
These simple changes add the capability of reading HCHO.
I should probably change the name (_tempo_l2_mm.py? _tempo_l2_mm_no2_hcho.py?), but I thought that I should do so directly when we abandon the _mm convention in next version of MONETIO.
I also took the oportunity to change the OrderedDict by a dict.
Cheers
Pablo

blychs committed:
Add capability to read HCHO

zmoon commented on monetio/sat/_tempo_l2_no2_mm.py:
```diff
@@ -102,13 +103,15 @@ def _open_one_dataset(fname, variable_dict):
             "temperature_profile",
+            "amf",
             "amf_total",
             "amf_diagnositc_flag",
```
Unless it's spelled wrong in the datasets
```suggestion
            "amf_diagnostic_flag",
```

zmoon commented:
> I should probably change the name (_tempo_l2_mm.py? _tempo_l2_mm_no2_hcho.py?), but I thought that I should do so directly when we abandon the _mm convention in next version of MONETIO.

Yeah let's leave it for now. Thanks.

blychs committed:
Update monetio/sat/_tempo_l2_no2_mm.py

You are right, that variable is not really used for now and I had not noticed.

Co-authored-by: Zachary Moon <zmoon92@gmail.com>
